homeassistant/helpers/config_validation.py

```diff
@@ -895,10 +895,7 @@ def validator(config: dict) -> dict:
         """Check if key is in config and log warning or error."""
         if key in config:
             try:
-                near = (
-                    f"near {config.__config_file__}"  # type: ignore[attr-defined]
-                    f":{config.__line__} "
-                )
+                near = f"near {config.__config_file__}:{config.__line__} "  # type: ignore[attr-defined]
```

Comment:
This line becomes 105 characters now. We try to keep them at max 88.

Reply:
Ok. This PR would address the issue, but really all that would be needed for the (eventual) update would be to add a second `type: ignore[attr-defined]`. Not that big of a deal IMO.

I can close this one if that works for you.

Reply:
Yeah, I think a second type ignore is ok.

We could probably tell mypy about these attributes if we would have better typing for the `config` parameter. So I hope these ignores are temporary.

Reply:
That probably depends on `voluptuous`, so not sure that will happen anytime soon, unfortunately. Personally I consider `type: ignore[attr-defined]` in a `try ... except AttributeError` block to be ok.